### PR TITLE
lower the parallelism when using core-test sats

### PR DESCRIPTION
**Pull request by alexcb**

### CODE REVIEW

**Comment by mikejholly** [comment]
Is this worth adding? It looks like most of the problem tests still failed and seemed to take a bit longer (one has nearly taken 1h).

File: .github/actions/stage2-setup/action.yml [hunk 1]
```diff
@@ -98,6 +98,8 @@ runs:
         if [ "$SATELLITE_NAME" = "core-test" ]; then
           SATELLITE_NAME="$SATELLITE_NAME-$(( $RANDOM % 4 ))"
           echo "Using core-test satellite: $SATELLITE_NAME"
+          # limit converstion parallelism to prevent overloading the core-test satellite
+          ${{inputs.SUDO}} ${{inputs.BUILT_EARTHLY_PATH}} config global.conversion_parallelism 2
```
Comment (vladaionescu) [inline .github/actions/stage2-setup/action.yml:102]: Couldn't we just limit this on the satellite side?

Comment (alexcb) [reply .github/actions/stage2-setup/action.yml:102]: we've tried that too, might make sense to just lower the parallelism again on the satellite size.

**Comment by alexcb** [comment]
lets just try lowering the satellite limit again

**Pull request closed without merging**